reflex/constants/base.py

```diff
@@ -24,7 +24,7 @@ class Dirs(SimpleNamespace):
     # The name of the utils file.
     UTILS = "utils"
     # The name of the output static directory.
-    STATIC = "_static"
+    STATIC = "out"
```

Comment:
is this rename necessary? personally, this will break my docker builds, github pages publishing.

deploying to hosting and using the .zip files still work tho

Reply:
yes it is, next JS changed the static folder to out for exporting